partial-response/src/main/java/com/iluwatar/partialresponse/FieldJsonMapper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Gopinath Langote
```

Comment:
Please keep the copyright unchanged e.g. Copyright (c) 2014-2017 Ilkka Seppälä. Check the other files for this too.

Reply:
@iluwatar : Updated copyright header to contain year range e.g. Copyright (c) 2014-2017 Gopinath Langote